cpp/src/c_api/cugraph_api.cpp

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cugraph_c/cugraph_api.h>
+#include <cugraph/api_helpers.hpp>
+
+#include <cugraph/visitors/rw_visitor.hpp>
+
+#include <cugraph/visitors/erased_api.hpp>
+
+#include <raft/handle.hpp>
+
+#include <rmm/device_buffer.hpp>
+#include <rmm/device_uvector.hpp>
+
+namespace helpers {
```

Comment:
No. I thought about that. Anonymous spaces aren't really anonymous (compiler generates names for them, too). But, more importantly, I want to collect all these "helpers" under a common...abstraction. So, different C++ sources pertaining to the C-API would enrich this same `namespace` rather than shovel them in the same `hpp` header, which then could not / should not define unnamed namespaces, anyway.